src/twisted/python/zippath.py

```diff
@@ -273,7 +273,7 @@ class ZipArchive(ZipPath[AnyStr, AnyStr]):
     _zipfileFilename: AnyStr
 
     @property
-    def archive(self):
+    def archive(self) -> Self:  # type: ignore[override]
```

Comment:
the parent type is a mutable `ZipArchive[_ArchiveStr]`

Reply:
Hmmmmmm. Ignore overrides typically mean a breakage of LSP. Is that happening here? I think the idea is that we want to conform to a more abstract FilePath protocol (which isn't specified because [`IFilePath` can't be generic](https://github.com/Shoobx/mypy-zope/issues/95) and `ZipArchive` and `ZipPath` are actually derivatives of that as-yet-unspecified thing?

If I'm interpreting this correctly can you file a separate issue, and maybe drop a comment on this line

Reply:
I don't know, I made a new issue to investigate